src/dotnet/commands/dotnet-sln/LocalizableStrings.resx

```diff
@@ -162,4 +162,7 @@
   <data name="ProjectsHeader" xml:space="preserve">
     <value>Project(s)</value>
   </data>
+  <data name="AddProjectSolutionFolderArgumentDescription" xml:space="preserve">
+    <value>The destination solution folder to add the items to</value>
```

Comment:
```suggestion
    <value>The destination solution folder path to add the project(s) to.</value>
```

Reply:
Ah, I just saw the issue you linked to regarding the future use of adding solution items via `dotnet sln add`.  I think we should go with "project" rather than "item" for now; we can always change this if we support arbitrary solution items via this command in the future.